Conceptpower+Spring/src/main/java/edu/asu/conceptpower/wrapper/ConceptEntryWrapper.java

```diff
@@ -17,6 +19,8 @@
  * @author Julia Damerow
  * 
  */
+@Component("conceptEntryWrapper")
+@Scope(value = "prototype")
 public class ConceptEntryWrapper implements Serializable {
```

Comment:
I was so sure I responded to that. Hm, well... apparently not.

Yes, remove component and create a new object with new.